src/strings/ops.c

```diff
@@ -557,8 +557,8 @@ MVMString * MVM_string_repeat(MVMThreadContext *tc, MVMString *a, MVMint64 count
         return a;
     if (count < 0)
         MVM_exception_throw_adhoc(tc, "repeat count (%"PRId64") cannot be negative", count);
-    if (count > (1 << 30))
-        MVM_exception_throw_adhoc(tc, "repeat count > %d arbitrarily unsupported...", (1 << 30));
+    if (count > 0xFFFFFFFF)
```

Comment:
Would be nice to have a named constant for this magic number. Especially when it's used in multiple places.